DFW.cpp

```diff
@@ -136,172 +206,128 @@ int DFW::joysticklh(void) // returns joystick right vertical
 int DFW::start(void) // returns start button state 

```

Comment:
All of the button functions should return bool (true meaning depressed, false otherwise).